server/command.go

```diff
@@ -103,7 +91,34 @@ func (p *Plugin) executeStartMeetingCommand(c *plugin.Context, args *model.Comma
 }
 
 func (p *Plugin) executeHelpCommand(c *plugin.Context, args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
-	text := "###### Mattermost Jitsi Plugin - Slash Command Help\n" + strings.Replace(commandHelp, "|", "`", -1)
+	l := p.getUserLocalizer(args.UserId)
+	helpTitle := l.MustLocalize(&i18n.LocalizeConfig{
```

Comment:
I worried about the use of `MustLocalize`. It tends to panic for often then one thinks. A helper method like the follow could be used to properly check for errors:
```go
// LocalizeDefaultMessage localizer the provided message
func (p *Plugin) LocalizeDefaultMessage(l *i18n.Localizer, m *i18n.Message) string {
	s, err := l.LocalizeMessage(m)
	if err != nil {
		p.API.LogWarn("Failed to localize message", "message ID", m.ID, "error", err.Error())
		return ""
	}

	return s
}
```

Reply:
This should panic only in cases where you are using wrongly the translations, something like passing a wrong plural argument or something inconsistent. I think is the right thing to do in that cases, just panic.

Reply:
Would such panic be caught in (release testing)? Can we ensure that such error is not likely to happen in a production env?

Reply:
Done